app/gui2/src/stores/graph/index.ts

```diff
@@ -327,11 +327,12 @@ export const useGraphStore = defineStore('graph', () => {
     const node = db.nodeIdToNode.get(id)
     if (!node) return
     edit((edit) => {
-      edit.getVersion(node.rootExpr).syncToCode(content)
+      const editExpr = edit.getVersion(node.innerExpr)
+      editExpr.syncToCode(content)
       if (withImports) {
         const conflicts = addMissingImports(edit, withImports)
         if (conflicts == null) return
-        const wholeAssignment = edit.getVersion(node.rootExpr)?.mutableParent()
+        const wholeAssignment = editExpr.mutableParent()
```

Comment:
I really don't understand the interaction of `getVersion` and `syncToCode` here. What caused the original issue? Why calling `getVersion` again causes things to break, especially since its return value is not really used other than for error reporting?

Reply:
Calling `getVersion` once is just a refactor. The functional change is using `node.innerExpr` instead of `node.rootExpr`.